iac-extensions/docker/src/test/java/org/sonar/iac/docker/parser/utils/Assertions.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.docker.parser.utils;
+
+import com.sonar.sslr.api.RecognitionException;
+import com.sonar.sslr.api.Rule;
+import com.sonar.sslr.api.typed.ActionParser;
+import java.nio.charset.StandardCharsets;
+import javax.annotation.Nullable;
+import org.fest.assertions.GenericAssert;
+import org.sonar.api.batch.fs.internal.DefaultTextPointer;
+import org.sonar.iac.docker.parser.DockerNodeBuilder;
+import org.sonar.iac.docker.parser.TreeFactory;
+import org.sonar.iac.docker.parser.grammar.DockerGrammar;
+import org.sonar.iac.docker.parser.grammar.DockerLexicalGrammar;
+import org.sonar.iac.docker.tree.api.DockerTree;
+import org.sonar.iac.docker.tree.impl.DockerTreeImpl;
+import org.sonar.sslr.grammar.GrammarRuleKey;
+import org.sonar.sslr.grammar.LexerlessGrammarBuilder;
+import org.sonar.sslr.tests.ParsingResultComparisonFailure;
+import org.sonar.sslr.tests.RuleAssert;
+import org.sonarsource.analyzer.commons.TokenLocation;
+
+public class Assertions {
+
+  public static RuleAssert assertThat(Rule actual) {
+    return new RuleAssert(actual);
+  }
+
+  public static ParserAssert assertThat(GrammarRuleKey rule) {
+    return assertThat(DockerLexicalGrammar.createGrammarBuilder(), rule);
+  }
+
+  public static ParserAssert assertThat(LexerlessGrammarBuilder b, GrammarRuleKey rule) {
+    return new ParserAssert(new ActionParser<>(
+      StandardCharsets.UTF_8,
+      b,
+      DockerGrammar.class,
+      new TreeFactory(),
+      new DockerNodeBuilder(),
+      rule));
+  }
```

Comment:
This can be simplified to:
```
  public static ParserAssert assertThat(GrammarRuleKey rule) {
    return new ParserAssert(new DockerParser(rule));
  }
```